activeadmin.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'railties',            '>= 4.0', '< 5.1'
   s.add_dependency 'ransack',             '~> 1.3'
   s.add_dependency 'sass-rails'
+  s.add_dependency 'thor',                '0.19.1', '!=0.19.2', '!=0.19.3', '!=0.19.4' # http://stackoverflow.com/questions/41207432/
```

Comment:
Is jbuilder a direct runtime dependency of AA or does this fix only a problem in our tests? If the latter, the dependency should be added to the Gemfile, or even better, to lock files committed to version control.